department-of-veterans-affairs/caseflow
# Added padding to reciept filter

Resolves [Sides of Receipt Date Dropdown Need Padding](https://jira.devops.va.gov/browse/APPEALS-29956)

# Description
The receipt filter has gained an additional 10px of padding on both the left and right sides. The recieptdate filter icon did not inherit these changes.
## Acceptance Criteria

## Testing Plan
1. Navigate to caseflow and choose a user that can access reader (BVADWISE works)
2. Navigate to the reader section of caseflow.
3. Click the reciept date filter Icon and notice that there is now additional padding that widens the reciept filter and gives the component some breathing room.

# Frontend
## User Facing Changes

 BEFORE|AFTER
<img width="199" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/d5c079b4-18f7-4eff-b5f1-4924c2360e0e">|<img width="214" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/f0553812-8c34-4776-aa65-ee66867a2ab2">
